peekingduck/pipeline/nodes/model/mask_rcnn.py

```diff
@@ -73,8 +73,8 @@ class Node(AbstractNode):  # pylint: disable=too-few-public-methods
         https://arxiv.org/abs/1703.06870
 
         Inference code adapted from:
-        https://pytorch.org/vision/stable/_modules/torchvision/models/detection/mask_rcnn.html
-        Model weights adapted from:
+        https://pytorch.org/vision/0.11/_modules/torchvision/models/detection/mask_rcnn.html
+        The weights for Mask-RCNN Model with ResNet50 FPN backbone were adapted from:
         https://download.pytorch.org/models/maskrcnn_resnet50_fpn_coco-bf2d0c1e.pth
```

Comment:
Consider adding a vertical whitespace so this is rendered on a new line. 

![image](https://user-images.githubusercontent.com/56420072/178879651-2c8e0dc7-86cc-468f-b410-216259179153.png)
Similar to how "Mask R-CNN:..." and "Inference code ..." are on separate line.

Reply:
amended